config/src/vespa/config/print/fileconfigformatter.cpp

```diff
@@ -128,7 +131,7 @@ struct ConfigEncoder : public ArrayTraverser,
         case vespalib::slime::OBJECT::ID: return encodeOBJECT(inspector);
         case vespalib::slime::NIX::ID: return;
         }
-        abort(); // should not be reached
+        LOG_ABORT("should not be reached"); // should not be reached
```

Comment:
Here I guess the abort could be removed as the compiler will warn if you do not handle all enumerations I think.

Reply:
no, this is switch on getId() which is uint32_t.